ginivision/src/androidTest/java/net/gini/android/vision/internal/camera/photo/PhotoTest.java

```diff
@@ -43,6 +45,37 @@ public void should_supportParceling() {
     }
 
     @Test
+    public void should_keepUserComment_whenCreating_fromDocument() {
```

Comment:
This is related to the UUID fix, but is also needed for the rotation delta.

The Document class contains the image and is part of the public API. The Photo class is used only internally.

The Screen API uses the Component API so we share the image in a Document instance between the Fragments of the Component API and then convert it back to a Photo instance internally.